spec/validations/manifest_spec.rb

```diff
@@ -256,12 +256,31 @@ def create_package(parameter_hash)
     end
   end
 
+  context 'location references a non-string URL' do
+    before do
+      @manifest_hash = {
+        'location' => {
+          'zendesk' => {
+            'ticket_sidebar' => {
+              'url' => true
+            }
+          }
+        }
+      }
+    end
+
+    it 'should have a location error' do
+      expect(@package).to have_error(/location does not specify a URI/)
```

Comment:
This might be a bit pedantic but isn't it a URL we're looking for not a URI. Or would `urn:isbn:0-486-27557-4` be valid in this instance?